web/viewer.html

```diff
@@ -27,6 +27,8 @@
 <!--#if GENERIC || CHROME-->
     <meta name="google" content="notranslate">
 <!--#endif-->
+    <!-- Ignore legacy IE compatibility settings and prefer HTML5 -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
I still need to test this change on Edge, and I.E. 9.0 and 10.0.  Will commit once this is done. 


     On Tuesday, September 1, 2015 8:10 AM, Jonas Jenwald <notifications@github.com> wrote:
   

 In web/viewer.html:> @@ -27,6 +27,8 @@
>  <!--#if GENERIC || CHROME-->
>      <meta name="google" content="notranslate">
>  <!--#endif-->
> +    <!-- Ignore legacy IE compatibility settings and prefer HTML5 -->
> +    <meta http-equiv="X-UA-Compatible" content="IE=edge">
Please also address the review comments in #6374 (comment), i.e. change the patch to:<!--#if GENERIC-->
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
<!--#endif-->After this, please squash the commits, see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits.—
Reply to this email directly or view it on GitHub.